src/components/posts/PostList.js

```diff
@@ -15,17 +18,69 @@ export const PostList = props => {
         getPosts()
     }, [])
 
+    const history = useHistory()
+
+    const handleDelete = postId => {
+      deletePost(postId)
+        .then(getPosts)
+    };
+
     return (
         <div className="postList">
-          <h1 className="text-center my-4">All Posts</h1>
+          <Row className="align-items-center my-4">
+            <Col>
+              <input placeholder="search" />
```

Comment:
I don't think we should include non-functioning components in the site. I would suggest waiting until this feature is built to add it.